src/pages/view_article.jsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getArticleById } from "../../utils/api";
+
+import * as React from "react";
+
+export default function View_Article() {
+  const [currentArticle, setCurrentArticle] = useState({});
+
+  const { article_id } = useParams();
+
+  useEffect(() => {
+    getArticleById(article_id).then((articleData) => {
+      setCurrentArticle(articleData);
+    });
+  }, [article_id]);
+
+  return (
+    <>
```

Comment:
This fragment doesn't affect the code as it all has one parent element anyway, `<main>